lib/core/src/server/dev-server.js

```diff
@@ -61,6 +61,11 @@ export default function (options) {
       devMiddlewareOptions
     );
 
+    router.get(/\/static\/media\/.*\..*/, (request, response, next) => {
+      response.set('Cache-Control', `public, max-age=31536000`);
```

Comment:
Ow I see, if there's a hash in the filename, then the hash changes when the file changes, so we can add a long term cache.

Reply:
How is this not applied to files without a hash?

Reply:
Essentially, we add this header only to resources under the `/static/media/` path; users don't really have the ability to directly put files there without them being put there by `file-loader` or `media-loader`. As a result, anything served from `/static/media` _will_ be a file that's hashed, so we can apply this header to any file served from here.

`file-loader` and `media-loader` are configured to use this path in [`manager-webpack.config.js:115`](https://github.com/storybookjs/storybook/blob/34ec239aa2b0ceb022d18e6c3e45ab8ca797faa7/lib/core/src/server/manager/manager-webpack.config.js#L115)